packages/client/src/store/resolvers.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * WordPress dependencies
+ */
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { ENTITY_KIND, ENTITY_NAME } from './constants';
+import { receiveAbilities } from './actions';
+import type { Ability } from '../types';
+
+/**
+ * Resolver for getAbilities selector.
+ * Fetches all abilities from the server with pagination.
+ */
+export function getAbilities() {
```

Comment:
Will there be a case when we want to invalidate the resolution for this resolver?